src/cript/api/api.py

```diff
@@ -194,6 +162,14 @@ def disconnect(self):
         global _global_cached_api
         _global_cached_api = self._previous_global_cached_api
 
+    @property
+    def api_handle(self):
+        return self._api_handle
+
+    @property
+    def api_version(self):
+        return self._api_version
```

Comment:
I don't think there is a scenario that either of these two will be needed and we'll always use only a single host

changing a single small string of `/api/v1/` is pretty simple that I wouldn't put any variables for it, although we could

Reply:
I wanted them as property, because one day we might need to do some magic to get it from back end instead of just reading a string.
And this way we have it read-only.

Reply:
this is pretty small change and when we need it in the future lets add it then because we might just add this and never need this or worst it doesn't help in the future at all

Reply:
k